### PR TITLE
updating mira endpoints

### PR DESCRIPTION
**Pull request by Tom-Szendrey**
# Description
They have updated epi dkg endpoint

https://github.com/DARPA-ASKEM/orchestration/pull/440

### CODE REVIEW

File: packages/client/hmi-client/src/services/concept.ts [hunk 1]
```diff
@@ -62,7 +62,7 @@ async function searchCuriesEntities(query: string): Promise<Array<DKG>> {
 }
 
 function getCurieUrl(curie: string): string {
-	return `http://mira-epi-dkg-lb-c7b58edea41524e6.elb.us-east-1.amazonaws.com/entity/${curie}`;
+	return `http://mira-epi-dkg-lb-dc1e19b273dedaa2.elb.us-east-1.amazonaws.com//entity/${curie}`;
```
Comment (dvince2) [inline packages/client/hmi-client/src/services/concept.ts:65]: Double `//`

File: packages/server/src/main/resources/application.properties [hunk 2]
```diff
@@ -148,7 +148,7 @@ terarium.amazon.s3.default.credentials-id=default
 ########################################################################################################################
 github.url=https://api.github.com
 jsdelivr.url=https://cdn.jsdelivr.net
-mira-api.url=http://mira-epi-dkg-lb-dc1e19b273dedaa2.elb.us-east-1.amazonaws.com
+mira-api.url=http://mira-epi-dkg-lb-dc1e19b273dedaa2.elb.us-east-1.amazonaws.com/
```
Comment (dgauldie) [inline packages/server/src/main/resources/application.properties:151]: you need the ending slash?

Comment (dvince2) [inline packages/server/src/main/resources/application.properties:151]: Also new `/` here

**Review by dgauldie — APPROVED** [review]

File: packages/client/hmi-client/src/services/concept.ts (hunk 1 above)
Comment (Tom-Szendrey) [reply packages/client/hmi-client/src/services/concept.ts:65]: Thank you so much

**Comment by dvince2** [comment]
The function in `concept.ts` is never called anywhere, and application.proeprties is unchanged. Do we actually just want to remove that function in `concept.ts`?

**Review by dvince2 — APPROVED** [review]

**Pull request merged**